### PR TITLE
[Doc] UX Icons doc tweaks

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Issues        | -
| License       | MIT

The docs contained some minor issues (understandable because the doc merge produced a ton of hard-to-solve conflicts 🙏 ). I fixed those and made some minor tweaks.

### CODE REVIEW

File: src/Icons/doc/index.rst [hunk 1]
```diff
@@ -89,40 +88,25 @@ Icon Sets
...
-`Iconoir`_                   1500 MIT        iconoir     iconoir:check
-`Ionicons`_                  1300 MIT        ion         ion:check
-`Lucide Icons`_              1500 MIT        lucide      lucide:check
-`Material Design Icons`_     5000 Apache 2   mdi         mdi:check
-`Octicons`_                   600 MIT        octicon     octicon:check
-`Phosphor Icons`_            7000 MIT        ph          ph:check
-`Tabler Icons`_              5200 MIT        tabler      tabler:check
-========================= ======= ========== =========== =================
-
-- `Bootstrap Icons <https://icons.getbootstrap.com/>`_
```
Comment (kbond) [inline src/Icons/doc/index.rst:114]: These links still need to exist for the table below, no?

**Review by smnandre — COMMENTED** [review]
Thank you Javier, sans sorry for the mess :(

File: src/Icons/doc/index.rst (hunk 1 above)
Comment (javiereguiluz) [reply src/Icons/doc/index.rst:114]: You are right ... the list of links should be fixed a bit, not removed. I re-added it at the bottom of the article. Thanks!

**Comment by smnandre** [comment]
Why is Doctor RST unhappy ?  :|

**Comment by javiereguiluz** [comment]
It was because of a silly mistake on my side. Fixed!

**Comment by kbond** [comment]
Thank you Javier.

**Pull request merged**